src/plugins/intel_npu/tests/functional/behavior/remote_tensor_tests/remote_run.hpp

```diff
@@ -76,12 +98,40 @@ TEST_P(RemoteRunTests, CheckRemoteTensorInternalBuf) {
     SKIP_IF_CURRENT_TEST_IS_DISABLED()
     ov::InferRequest inference_request;
 
-    OV_ASSERT_NO_THROW(compiled_model = core->compile_model(ov_model, target_device, configuration));
+    auto zero_context = core->get_default_context(target_device).as<ov::intel_npu::level_zero::ZeroContext>();
+    OV_ASSERT_NO_THROW(compiled_model = core->compile_model(ov_model, zero_context, configuration));
     OV_ASSERT_NO_THROW(inference_request = compiled_model.create_infer_request());
 
     auto tensor = inference_request.get_input_tensor();
+
+    auto remote_tensor =
+        zero_context.create_l0_host_tensor(ov::element::f32, tensor.get_shape(), ov::intel_npu::TensorType::INPUT);
+    tensor = {};
+
+    ov::Tensor check_remote_tensor;
+    ASSERT_NO_THROW(check_remote_tensor = remote_tensor);
```

Comment:
I suggest using `OV_ASSERT_NO_THROW` because it provides way more error info than this one.

Reply:
good catch, thanks!